# ğŸ§¬ Alastair Smith

**Postdoctoral Researcher**  
MRC Weatherall Institute of Molecular Medicine (WIMM)  
University of Oxford  

ğŸ”¬ [Milne Group â€“ Epigenetics & Gene Regulation in Leukaemia](https://www.imm.ox.ac.uk/research/research-groups/milne-group-epigenetics-and-gene-regulation-in-leukaemia)

ğŸ“ Oxford, UKâ€ƒ|â€ƒğŸ§ª Computational & Experimental Epigenomicsâ€ƒ|â€ƒğŸ’¡ Enhancer Biology in Cancer

---

## ğŸ”¬ Research Focus

I'm a postdoctoral researcher investigating **enhancer dynamics in cancer**, with a particular focus on **MLL-AF4 Acute Lymphoblastic Leukaemia (ALL)**. My work sits at the interface of **experimental epigenomics and computational biology**, aiming to unravel how chromatin architecture drives transcriptional dysregulation in leukaemia.

To address these questions, I generate and integrate diverse high-throughput datasets using:

- **Epigenomic profiling**: ChIP-seq, low-input ChIPmentation, ATAC-seq, CUT&Tag  
- **Transcriptomics**: Bulk RNA-seq and single-cell multiome  
- **Chromatin conformation**: NG Capture-C, Tiled-C  
- **Epigenetic modifications**: DNA methylation profiling
- **Single cell Transcriptomics and Epigenomics**: 10x scMultiome, scCUT&TAG

These data are analysed using custom workflows, statistical modelling, and machine learning to identify **regulatory mechanisms underpinning transcriptional dysregulation in leukemia**.

---

## ğŸ› ï¸ Software & Tools

I build open-source tools designed for **performance, reproducibility, and ease of use** in large-scale genomics.

- ğŸ”„ [**SeqNado**](https://github.com/alsmith151/SeqNado): Modular Snakemake pipelines for processing ChIP-seq, ATAC-seq, RNA-seq, CUT&Tag, WGS, CRISPR screens, and Micro-Capture-C (MCC) data  
- ğŸ§¬ [**CapCruncher**](https://github.com/sims-lab/CapCruncher): Toolkit for analysing NG Capture-C, Tri-C, and Tiled-C chromatin interaction data  
- âš¡ [**BamNado**](https://github.com/alsmith151/BamNado): A fast, Rust-based utility for efficient BAM file manipulation  
- ğŸ“Š [**PlotNado**](https://github.com/alsmith151/PlotNado): Command-line genomic visualisation in a genome browser-like style  

---

## ğŸ§  Technical Expertise

- **Languages**: Python, Rust, R, Bash  
- **Frameworks & Tools**: Snakemake, Conda, Docker, Git, Apptainer, AWS 
- **Domains**: Epigenomics, transcriptomics, chromatin architecture, multi-omics integration  
- **Focus Areas**: Reproducible workflows, NGS tool development, cancer epigenetics  

---

## ğŸ”— Contact & Profiles

- ğŸ”¬ [Milne Group @ Oxford](https://www.imm.ox.ac.uk/research/research-groups/milne-group-epigenetics-and-gene-regulation-in-leukaemia)  
- ğŸ§ª [ORCID](https://orcid.org/0000-0001-6330-1407)
- ğŸ“« [Email](alastair.smith@imm.ox.ac.uk )
- ğŸ’¼ [LinkedIn](https://www.linkedin.com/in/alastairlukesmith1992/)
